src/html/index.html

```diff
@@ -256,6 +262,8 @@ <h4 class="card-title">
   var uiManager;
   bitmovin.playerui.UIManager.setLocalizationConfig({ language: 'en', browserLanguageDetection: false });
 
+  bitmovin.player.Player.addModule(bitmovin.player["advertising-bitmovin"].default);
```

Comment:
Why the switch to the Bitmovin Ads module? The default is IMA, shouldn't the UI use the default player config?

Reply:
Because with IMA there is no control over the UI as IMA provides the minimal ads UI which cannot officially (and must not) be customized. With BAM this is possible to use our UI, so that made more sense to me.